File: Menu Driven Class Editor/Student.java
public class Student extends Person{
	private int year;
	Student(String name, String address, String phone, int year) {
		super(name, address, phone);
		this.year = year;
	}

	int getGraduationYear() {
		return this.year;
	}
	public void setGraduationYear(int year) {
		this.year = year;
	}
	public String toString() {
		String y = "Year: " + year;
		return super.toString() + "\n" + y;

	}
}

File: Linked List Inheritance/Student.py
from Person import Person

class Student(Person):
    def __init__(self,newName='none',address='none',phone='999-999-9999',year=9999):
        self.name = newName
        self.address = address
        self.phone = phone
        self.year = year
    
    def setGraduationYear(self,graduationYear):
        self.year = graduationYear
    
    def getGraduationYear(self):
        return self.year
    
    def __str__(self):
        return f'Name: {self.name}\nAddress: {self.address}\nPhone: {self.phone}\nYear: {self.year}'
